### ISSUE TITLE
nx 15.9.4 forces use of node 18

### ISSUE THREAD

**Issue by joshbeckerlegdandg** [nrwl/nx#18744]
### Current Behavior

Installing nx v15.9.4 with node 16.16.0 and npm 8.19.4 forces use of node 18. 

[This line ](https://github.com/nrwl/nx/blob/06c3e5c9e15882b67d255fe6cf22da747f93747f/package.json#L290C40-L290C40) results in `@yarnpkg/parsers@3.0.0-rc.49` being installed as a dependency.

Which causes the error

```
npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: @yarnpkg/parsers@3.0.0-rc.49
npm ERR! notsup Not compatible with your version of node/npm: @yarnpkg/parsers@3.0.0-rc.49
npm ERR! notsup Required: {"node":">=18.12.0"}
npm ERR! notsup Actual:   {"npm":"8.19.4","node":"v16.16.0"}
```

### Expected Behavior

nx 15.9.4 should install correctly with node v16 as per [nx documentation](https://nx.dev/packages/workspace/documents/nx-nodejs-typescript-version-matrix)

### Steps to Reproduce

1. use node v16
2. create nx angular workspace version 15.9.4
3. npm install

### Nx Report

```shell
>  NX   Report complete - copy this into the issue template

   Node : 16.16.0
   OS   : win32 x64
   npm  : 8.19.4
   
   nx (global)             : 16.5.1
   nx                      : 15.9.4
   @nrwl/js                : 15.9.4
   @nrwl/jest              : 15.9.4
   @nrwl/linter            : 15.9.4
   @nrwl/workspace         : 15.9.4
   @nrwl/angular           : 15.9.4
   @nrwl/cli               : 15.9.4
   @nrwl/cypress           : 15.9.4
   @nrwl/devkit            : 15.9.4
   @nrwl/eslint-plugin-nx  : 15.9.4
   @nrwl/storybook         : 15.9.4
   @nrwl/tao               : 15.9.4
   @nrwl/web               : 15.9.4
   @nrwl/webpack           : 15.9.4
   typescript              : 4.9.5
   ---------------------------------------
   Community plugins:
   @storybook/angular : 7.3.2
```

### Operating System

- [ ] macOS
- [ ] Linux
- [X] Windows
- [ ] Other (Please specify)

**Comment by AgentEnder** [comment]
Hey @joshbeckerlegdandg 👋, this has already been addressed in recent Nx versions (it looks like it was addressed in a 16.4.x release).

I've chatted with @FrozenPandaz about if that patch should be backported and we will do so, but in the meantime you can likely force the resolution by specifying @yarnpkg/parsers@3.0.0-rc.46 manually in your package.json or using an override.

**Comment by joshbeckerlegdandg** [comment]
Hey @AgentEnder thanks for the help, will use the workaround
